src/views/preview/comment/comment.jsx

```diff
@@ -113,6 +158,24 @@ class Comment extends React.Component {
                         </FlexRow>
                     ) : null}
                 </FlexRow>
+                {this.state.deleting ? (
+                    <DeleteCommentModal
+                        isOpen
+                        key="delete-comment-modal"
+                        onDelete={this.handleConfirmDelete}
+                        onReport={this.handleConfirmReport}
+                        onRequestClose={this.handleCancelDelete}
+                    />
+                ) : null}
+                {this.state.reporting || this.state.reportConfirmed ? (
```

Comment:
for clarity, could you put parentheses around `this.state.reporting || this.state.reportConfirmed` ? Not sure about other folks, but that helps me feel confident of the way the order of operations is being handled

Reply:
Fixed in 237c6ec